javascript/builtins/Intl.json

```diff
@@ -61,7 +61,7 @@
                 "version_added": "24"
               },
               "chrome_android": {
-                "version_added": "26"
+                "version_added": "25"
```

Comment:
This applies to WebView as well. So "≤37".

Reply:
WebView was explicitly set to `false` for this feature, but I would assume that there's full parity between the JavaScript features?